can/uwrt_mars_rover_can_2022.yaml

```diff
@@ -0,0 +1,14 @@
+uwrt_mars_rover_can_bus:
+  bus_frequency: 1000000
+  canid_filter_mask: 2016
+  canid_filters:
+    1824: COMMON_FILTER
```

Comment:
what does the common filter include? maybe yaml comments for what these filtered values are supposed to encompass